include/common.h

```diff
@@ -1,14 +1,20 @@
-#pragma once
+﻿#pragma once 
 #include <random>
 #include <chrono>
 #include <cmath>
 
+#define GLM_ENABLE_EXPERIMENTAL
```

Comment:
If this is needed to build, you're likely using GLM 0.9.9. This version is actually a huge breaking change from GLM 0.9.8, as in this version vectors are not default initialized to 0 and matrices are not default initialized to identity. I've definitely left a bunch of uninitialized vectors/matrices around that will likely break a lot of things. For now, I'd just use GLM 0.9.8 until I can get around to fixing this.